ios/PurchasesHybridCommon/PurchasesHybridCommon/StoreProduct+HybridAdditions.swift

```diff
@@ -12,7 +12,11 @@ import StoreKit
 
 @objc public extension StoreProduct {
     
-    // Re-exports pricePerPeriod properties with different names to avoid recursion.
+    // Re-exports price properties with different names to avoid recursion.
+
+    @objc(price) var priceAmount: NSDecimalNumber {
```

Comment:
Hmm I noticed that the underlying property is `priceDecimalNumber` so couldn't we just expose it in objc as `priceAmount`? (Not sure if there is other benefit to change the name in objc to `price`)
```suggestion
    @objc var priceAmount: NSDecimalNumber {
```

Reply:
This was inspired by purchases-ios, which also renames `priceDecimalNumber` to `price` for ObjC, see [here](https://github.com/RevenueCat/purchases-ios/blob/main/Sources/Purchasing/StoreKitAbstractions/StoreProduct.swift#L206). Do you think there are potential downsides to this renaming?

Reply:
Mostly that there is a mismatch between the names in swift and objc, which can be confusing, specially when comparing different SDKs. As in, if we're going to be doing this, I would suggest to also rename the `priceAmount` in swift to `price` (though I very slightly prefer `priceAmount`?).

Reply:
Yea no you're right, I've also started to prefer `priceAmount`. `price` is too ambiguous. (E.g. is it formatted, is it the amount?) https://github.com/RevenueCat/purchases-hybrid-common/pull/911/commits/9f94b1745fca341794d5d6e082c8023c08d1afc7